tests/framework/utils.py

```diff
@@ -82,10 +85,10 @@ def __init__(self, docker_image_dict) -> None:
                 raise Exception(f"Image {key} does not exist!")
         self.docker_image_dict = docker_image_dict
 
-    def prepare_operator(self):
+    def prepare_operator(self, namespace = 'default', patch = jsonpatch.JsonPatch([])):
```

Comment:
For me, it is better for `namespace` to be a class member rather than an argument of `prepare_operator`. Because `OperatorManager` controls the lifecycle of KubeRay operator, it may need to implement a new function `delete_operator` which requires `namespace` in the future.

Reply:
Maybe we also need to change `patch` to a class member too.